tests/unit/test_handler.py

```diff
@@ -334,11 +352,17 @@ def test_parse_omit(code_str, code_list):
             (mock_other_param, expected_mips_dict_processed_other),
             (mock_inactive_param, expected_mips_dict_processed_inactive),
             (mock_no_program_param, expected_mips_dict_processed_no),
-            (mock_other_param | mock_no_program_param, expected_mips_dict_processed_other_no),
+            (mock_priority_param, expected_mips_dict_processed_priority_codes),
```

Comment:
is there a test where priority_codes contains a code that is not in the list of returned codes?

Reply:
no but that's a good idea, I'll add one